Engine Hacks/Skill System/Other Skills/armsthrift.s

```diff
@@ -50,6 +51,25 @@ NonMiss:
 NonArmsthrift:
 	@ ACTUAL ARMSTHRIFT CHECK END
 
+	@2wb: begin AxeFaith check
```

Comment:
Please change the name of the file to reflect that it's both doing the armsthrift and axefaith check (This does involve changing the installer too, note)